oraichain/assetlist.json

```diff
@@ -50,6 +50,524 @@
           }
         }
       ]
+    },
+    {
+      "description": "The ATOM native token of Oraichain.",
+      "denom_units": [
+        {
+          "denom": "ibc/A2E2EEC9057A4A1C2C0A6A4C78B0239118DF5F278830F50B4A6BDD7A66506B78",
+          "exponent": 0
+        },
+        {
+          "denom": "atom",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/A2E2EEC9057A4A1C2C0A6A4C78B0239118DF5F278830F50B4A6BDD7A66506B78",
+      "name": "ATOM",
+      "display": "atom",
+      "symbol": "ATOM",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "cosmoshub",
+            "base_denom": "uatom",
+            "channel_id": "channel-301"
+          },
+          "chain": {
+            "channel_id": "channel-15",
+            "path": "transfer/channel-15/uatom"
+          }
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/cosmoshub/images/atom.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/cosmoshub/images/atom.svg"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/cosmoshub/images/atom.png",
```

Comment:
Can we add in image_sync for all images not in oraichain's /images/ direrctory, please? It's usually same counterparty data as the trace
```
"image_sync": {
  "chain_name": "cosmoshub",
  "base_denom": "uatom"
}
```